src/genlib_retry.erl

```diff
@@ -12,10 +14,20 @@
 
 -export([next_step/1]).
 
+-export_type([policy/0]).
+
 -export_type([strategy/0]).
```

Comment:
Then maybe rename this type to simple plain `t()` as it conflicts with `policy()` semantically?